articles/custom-domains/additional-configuration.md

```diff
@@ -103,6 +108,21 @@ If you want to use social identity providers with your custom domain, you must u
 ::: warning
 You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
 :::
+  
+## Configure Google Apps connections
```

Comment:
This is all good but we must add this info also at the corresponding docs. For example, add this content at this doc https://auth0.com/docs/connections/enterprise/google-apps as an optional step if you are using custom domains.